Makefile

```diff
@@ -48,8 +35,6 @@ coverage:
 	xcrun llvm-cov export -format="lcov" .build/debug/mongo-swift-driverPackageTests.xctest/Contents/MacOS/mongo-swift-driverPackageTests -instr-profile .build/debug/codecov/default.profdata > info.lcov
 
 clean:
-	rm -rf Packages
```

Comment:
this might have been relevant long ago when the driver supported cocoapods, but isn't now